bothub/api/grpc/repository/services.py

```diff
@@ -0,0 +1,17 @@
+from django_grpc_framework import mixins
+from django_grpc_framework import generics
+
+from .serializers import RepositoryProtoSerializer
+from bothub.common.models import Repository
+
+
+class RepositoryService(mixins.ListModelMixin, generics.GenericService):
+    queryset = Repository.objects.all()
+    serializer_class = RepositoryProtoSerializer
+
+    def filter_queryset(self, queryset):
+        owner_id = self.request.owner_id
+        if owner_id:
+            queryset = queryset.filter(owner__pk=owner_id)
```

Comment:
This filter has to consider that an organization may be allowed in some intelligence, not only owner, but with permission, because there are cases of not being owner but having permission in some intelligence